README.md

```diff
@@ -26,11 +26,20 @@ Demonstrate your understanding of this Sprint's concepts by answering the follow
 
 - [ ] What is React JS and what problems does it try and solve? Support your answer with concepts introduced in class and from your personal research on the web.
 
+  React JS is a javascript library that is made up of HTML and uses components. React helps greatly with scaling through the use of components. 
+
+
+
 - [ ] What does it mean to _think_ in react?
 
+    Thinking react means thinking in terms of how code can be re-used in the future. Seeing which parts of your page can be broken down into components.
```

Comment:
Yes. There's more. Remember that article? https://reactjs.org/docs/thinking-in-react.html 

To think in React means planning your React application in a specific way so that the data is managed from the top down. It means to break your UI into components, create a static wireframe, and then build out your user interface and logic functions from there. :)